src/python/espressomd/particle_data.pyx

```diff
@@ -2026,8 +2048,7 @@ Set quat and scalar dipole moment (dipm) instead.")
         # Ids of the selected particles
         ids = []
         # Did we get a function as argument?
-        if len(args) == 1 and len(kwargs) == 0 and isinstance(
-                args[0], types.FunctionType):
+        if len(args) == 1 and len(kwargs) == 0 and callable(args[0]):
```

Comment:
remark: classes are callable

Reply:
Didn't find an other way to handle Cython oddities.